services/src/machine_learning/xgboost_training.rs

```diff
@@ -0,0 +1,115 @@
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use typetag::serde;
+
+use serde_json::Value;
+use snafu::{ensure, ResultExt};
+use xgboost_rs::{Booster, DMatrix};
+
+use crate::error::Result;
+use crate::machine_learning::xg_error as XgModuleError;
+use crate::machine_learning::MachineLearningFeature;
+use ordered_float::NotNan;
+
+use super::MachineLearningAggregator;
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct XgboostTrainingParams {
+    pub no_data_value: f32, // FIXME: remove?
```

Comment:
add comment for why it should be removed

Reply:
yes, there is no nodata value, so the user will have to select one on their own, right? Then you can remove the FIXME comment and instead document what the parameter is

Reply:
Ok, I guess this could be used to train a model while ignoring a specific input value.

20ed2987

commit msg:
chore: add comment to no_data_value in xbgoost training params